data/json/items/comestibles/med.json

```diff
@@ -550,15 +542,7 @@
             "type" : "consume_drug",
             "activation_message" : "You shoot up.",
             "tools_needed" : { "syringe" : 1 },
-            "effects" : [ {
-                    "id" : "pkill3",
-                    "duration" : 80
-                },
-                {
-                    "id" : "pkill2",
-                    "duration" : 200
-                }
-            ]
+            "effects" : [ { "id" : "pkill3", "duration" : 80 }, { "id" : "pkill2", "duration" : 200 } ]
```

Comment:
No, split these similar to the other examples within the same file that have multiple effects